src/store/audio.ts

```diff
@@ -1045,41 +1015,16 @@ export const audioStore = createPartialStore<AudioStoreTypes>({
     },
   },
 
-  APPLY_AUDIO_PRESET_TO_AUDIO_ITEM: {
-    // FIXME: audioItemを直接書き換えないようにするか、関数化する
-    mutation(state, { audioItem }) {
-      if (
-        audioItem == undefined ||
-        audioItem.presetKey == undefined ||
-        audioItem.query == undefined
-      )
-        return;
-      const presetItem = state.presetItems[audioItem.presetKey];
-      if (presetItem == undefined) return;
-
-      // Filter name property from presetItem in order to extract audioInfos.
-      const { name: _, morphingInfo, ...presetAudioInfos } = presetItem;
-
-      // Type Assertion
-      const audioInfos: Omit<
-        AudioQuery,
-        "accentPhrases" | "outputSamplingRate" | "outputStereo" | "kana"
-      > = presetAudioInfos;
+  APPLY_AUDIO_PRESET: {
+    mutation(state, { audioKey }: { audioKey: AudioKey }) {
+      const audioItem = state.audioItems[audioKey];
 
-      audioItem.query = { ...audioItem.query, ...audioInfos };
+      if (!audioItem || !audioItem.presetKey) return;
```

Comment:
`audioItem`がない場合は想定外なのでエラーにするのが一応正しいかなと思いました。
まあどちらでもあまり変わらないので、気が向けば･･･！
```suggestion
      if (!audioItem.presetKey) return;
```

Reply:
おっと、 `AudioItem | undefined` でなくなったのでここもさくっと対応すべきでした 🙇 
suggestion取り込ませていただきました！